Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -670,6 +778,12 @@ public function removeTrailingWhitespace($index, array $opts = array())
      */
     public function setCode($code)
     {
+        // No need to work when the code is the same.
+        // That is how we avoid a lot of work and setting changed flag.
+        if ($code === $this->generateCode()) {
+            return;
+        }
```

Comment:
it will avoid reparsing the code (and retransforming it), so it will make things a bit faster here AFAIK, no ?